templates/nextjs/gitignore

```diff
@@ -35,11 +35,6 @@ next-env.d.ts
 
 .fuels
 
-src/sway-api/contracts
-src/sway-api/predicates
-src/sway-api/scripts
-src/sway-api/index.ts
-
```

Comment:
Is there any value in having these as a comment?

```bash
# Uncomment the lines below if you don't wish
# to commit the generated files to git.

# src/sway-api/contracts
# src/sway-api/predicates
# src/sway-api/scripts
# src/sway-api/index.ts
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3175/commits/34bb78f9b62c05f1717d41c4d5e0f04d331e6665